www/artists/ihauser/index.html

```diff
@@ -39,7 +39,7 @@ <h3>
         </div>
         <div id="footer">
             <h6>
-                © Copyright 2013. All images are the property of the artist. Coded 2013 Mykola Kyba. Updated 2015 Bennett Eidsness. Expanded Images 2018 Ryan Robertson
+                &copy Copyright 2013. All images are the property of the artist. Coded 2013 Mykola Kyba. Updated 2015 Bennett Eidsness. Expanded Images 2018 Ryan Robertson
```

Comment:
The only actual change that is needed for this pull request. Literally everything else is completely irrelevant, and _shouldn't be here._